kitchen-ec2.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "test-kitchen", "~> 1.4", ">= 1.4.1"
   gem.add_dependency "excon"
   gem.add_dependency "multi_json"
-  gem.add_dependency "aws-sdk", "~> 2"
+  gem.add_dependency "aws-sdk", "~> 3"
```

Comment:
This needs to be x-ref'd with ChefDK.

Reply:
ChefDK is using ~> 2, I am going to change it back.